.changeset/odd-files-attend.md

```diff
@@ -0,0 +1,6 @@
+---
+"@wpengine/wp-graphql-content-blocks": major
+---
+
+Fix: use `use_block_editor_for_post_type` instead of `post_type_supports` when filtering the post types.
+WARNING: Potential BREAKING schema changes on previously exposed blocks that do not support the block editor. Those blocks will no longer inherit the `editorBlocks` field.
```

Comment:
```suggestion
**BREAKING**: Potential schema changes on previously exposed blocks that do not support the block editor. Those blocks will no longer inherit the `editorBlocks` field.
```

Small nitpick, but let's call out "breaking" at the beginning of the line in the case someone quickly skims the release notes before updating.